doc/rbd/index.rst

```diff
@@ -4,18 +4,17 @@
 
 .. index:: Ceph Block Device; introduction
 
-A block is a sequence of bytes (often 512).
-Block-based storage interfaces are a mature and common way to store data on
-media including HDDs, SSDs, CDs, floppy disks, and even tape.
-The ubiquity of block device interfaces is a perfect fit for interacting
-with mass data storage including Ceph.
-
 Ceph block devices are thin-provisioned, resizable, and store data striped over
-multiple OSDs.  Ceph block devices leverage
-:abbr:`RADOS (Reliable Autonomic Distributed Object Store)` capabilities
-including snapshotting, replication and strong consistency. Ceph block
-storage clients communicate with Ceph clusters through kernel modules or
-the ``librbd`` library.
+multiple OSDs. Ceph block devices leverage :abbr:`RADOS (Reliable Autonomic
+Distributed Object Store)` capabilities including snapshotting and replication,
+and they provide strong consistency. Ceph block storage clients communicate
+with Ceph clusters by means of kernel modules or the ``librbd`` library.
```

Comment:
```suggestion
with Ceph clusters by means of a Linux kernel module or the ``librbd`` library.
```

Reply:
I take your point, but I'd like to beef up the paragraph that defines block storage and completely rewrite the last sentence. I'll chew on it and make sure you guys are satisfied before we merge anything here.